### PR TITLE
Us001 view roles

### PR DESCRIPTION
**Pull request by CallumE74**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,7 +5,9 @@
   "main": "dist/app.js",
   "scripts": {
     "start": "npm run build && node dist/app.js",
-    "build": "npx tsc && cp -R src/views dist"
+    "build": "npx tsc && cp -R src/views dist",
+    "test-ui": "mocha --require ts-node/register --timeout 10000 test/ui/*.ts",
+    "test": "mocha --require ts-node/register test/unit/**/*.ts"
```
Comment (Adaray786) [inline package.json:10]: Add how to run the tests in the ReadMe

File: src/service/JobRoleService.ts [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import {JobRole} from "../model/JobRole"
+
+const axios = require('axios');
+
+module.exports.viewJobRoles = async function (): Promise<JobRole []> {
+    try{
+        const response = await axios.get('http://localhost:8080/api/job-roles')
```
Comment (shaunganley) [inline src/service/JobRoleService.ts:7]: don't hardcode base URL, pass via ENV VAR

File: test/ui/JobRoleUITests.ts [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+const chaiInstance = require('chai');
+
+describe('JobRole Tests', async () => {
+    describe('View JobRoles', async () => {
+        it('View job role table', async () => {
+            var driver = new webdriver.Builder().
+            withCapabilities(webdriver.Capabilities.chrome()).
+            build();
+
+            await driver.get('http://localhost:3000/ViewRoles')
```
Comment (shaunganley) [inline test/ui/JobRoleUITests.ts:12]: don't hardcode base URL, pass via ENV VAR

**Review by shaunganley — CHANGES_REQUESTED** [review]

**Review by shaunganley — APPROVED** [review]
approved but looks like the build isn't running on this PR, might need rebased with main

**Review by Adaray786 — APPROVED** [review]
lgtm

**Pull request merged**